inspirehep/dojson/hep/fields/bd20x24x.py

```diff
@@ -32,7 +32,6 @@
 
 
 @hep.over('title_translations', '^242[10_][0_]')
```

Comment:
Because I think we need to keep titles in English and the translated version in separate fields, so that we can apply different mappings in ES. I know it's making the schema unnecessarily complex, but I don't see how I would be able e.g. to apply a specific language dependent stemming without separating the fields.